Core/src/OffConverter/OffReader.h

```diff
@@ -26,24 +26,83 @@ along with this program. If not, see <http://www.gnu.org/licenses/>.
 #include "OffModel.h"
 
 OIP_NAMESPACE_OPENINFRAPLATFORM_CORE_OFFCONVERTER_BEGIN
+
+/*! \brief The Off reader class
+	*
+	* This class is used to read the information from an .off file and pass it to the Off Model
+	*/
```

Comment:
```suggestion
/*! \brief The Off reader class
 *
 * This class is used to read the information from an .off file and pass it to the Off Model
 */
```

We try to have the starts vertically aligned in Doxycomments.

Reply:
Resolved: 464a368